editors/code/package.json

```
@@ -323,6 +323,14 @@
                     "default": true,
                     "markdownDescription": "Check all targets and tests (will be passed as `--all-targets`)"
                 },
+                "rust-analyzer.checkOnSave.noDefaultFeatures": {
+                    "type": [
+                        "null",
+                        "boolean"
+                    ],
+                    "default": null,
+                    "markdownDescription": "Do not activate the `default` feature"
```

```suggestion
                    "markdownDescription": "Do not activate the default features"
```

And maybe you could change the other one too?

---

```
        --no-default-features                        Do not activate the `default` feature
```

Oh...

---

Shouldn't we change this in another PR? This is not really the point of this PR.
In my opinion, the way it was previously phrased, is more clear anyway.

---

Yeah, don't mind me. I guess it's all right, since that's what `cargo` says.

On the other hand, there's no `default` feature:

```
cargo check --features default
error: Package `hello v0.1.0 (~/hello)` does not have these features: `default`
```

---

https://doc.rust-lang.org/cargo/reference/features.html

---

Well, yes:

> Default features could be excluded using --no-default-features.

So `--no-default-features` isn't about a feature called `default` (as the help text implies), but about not enabling the default features.

---

But you define the default features via a feature called `default` (https://doc.rust-lang.org/cargo/reference/features.html#the-features-section)